modules/core/05-port/keeper/keeper.go

```diff
@@ -4,17 +4,20 @@ import (
 	"context"
 	"strings"
 
+
 	"cosmossdk.io/log"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/cosmos/ibc-go/v9/modules/core/05-port/types"
+	"github.com/cosmos/ibc-go/v9/modules/core/api"
 	"github.com/cosmos/ibc-go/v9/modules/core/exported"
 )
 
 // Keeper defines the IBC connection keeper
 type Keeper struct {
-	Router *types.Router
+	Router   *types.Router
+	RouterV2 *api.Router
```

Comment:
not sure where this should actually go, we can figure this out later unless someone has a strong opinion about putting it somewhere now!